### ISSUE TITLE
Make Input Delay compensation configurable

### ISSUE THREAD

**Issue by Blebowski** [Blebowski/iso-16845-compliance-tests#4]
Several tests have limitations due to input delay. BRP or BRP_FD must be set to at least 2 or 3 to make the test passing.
Currently, some tests do have compensation scheme, however, not bit timing tests, these have over-written BRP,BRP_FD
by hand. 

It would be good to add compensation scheme in the tests that would allow running any test on any bit-rate.

Note that the tests that freeze BRP and BRP_FD to at least 2 or 3 are not functionally affected by this change.
In other words, they still validly test the use-case, they only fail at high bit-rates due to imperfections of the
test environment.
